serving/samples/helloworld-scala/build.sbt

```diff
@@ -0,0 +1,55 @@
+enablePlugins(JavaAppPackaging, DockerPlugin)
+
+organization := "Scala & Akka Examples for Knative Serving"
+
+name := "helloworld-scala"
+
+version := "0.0.1"
+
+scalaVersion := "2.12.8"
```

Comment:
Does this support some form of semver comparison, e.g. `">=2.12.8,<3"`?

Reply:
Yes and no, it is the scalaVersion used to build the project, but it will be bincompat forward-backward with all 2.12 releases.

Reply:
Ah, so it will use the newest version which is compatible in the dependency chain of the project. But this is the language dependency (just like you can't depend on specific java versions in Maven/ANT for instance).